cds_migrator_kit/videos/weblecture_migration/users/runner.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2025 CERN.
+#
+# cds-migrator-kit is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""cds-migrator-kit migration streams runner."""
+
+from pathlib import Path
+
+import yaml
+from invenio_rdm_migrator.streams import Stream
+
+from cds_migrator_kit.videos.weblecture_migration.logger import SubmitterLogger
+
+
+class VideosSubmitterRunner:
```

Comment:
For the future, we could have abstracted this class as well, rather than duplicate it